tendrl/gluster_integration/manager/__init__.py

```diff
@@ -34,10 +34,12 @@ def main():
     NS.state_sync_thread = sds_sync.GlusterIntegrationSdsSyncStateThread()
 
     NS.node_context.save()
-    NS.tendrl_context.save()
-    NS.gluster_integration.definitions.save()
-    NS.gluster_integration.config.save()
+    NS.gluster.tendrl_context.save()
```

Comment:
I tried using `NS.tendrl_context.save()` only in manager now, but the issue is from this `save()` function when it tries to load `tendrl_context`, its an empty object and doesn't contain `integration_id` populated. Thus the whole issue crops up again back and cluster details are written like `clusters//Volumes` etc to central store.
Due to this I feel in `gluster-integration` we do need logic to load `tendrl_context` from local which should be written while import/create cluster flow. Otherwise from `gluster-integration` when we initialize `NS.tendrl_context`, it should rather read details from etcd and assign to NS somehow.
@r0h4n comments?